lib/__tests__/is-visible.ts

```diff
@@ -51,6 +51,25 @@ describe('assert.dom(...).isVisible()', () => {
     });
   });
 
+  describe('with Element', () => {
+    test('fails for missing element', () => {
+      document.body.innerHTML = '<h1 class="baz">foo</h1>bar';
+
+      const headingElement = document.querySelector('h2');
+      console.log(headingElement);
+      assert.dom(headingElement).isVisible();
```

Comment:
I guess we could just pass `undefined` directly here instead of doing the `querySelector()` dance above

Reply:
We totally could. I guess this was existing?

Reply:
I think passing `null` is more appropriate, as that's what's returned if an element isn't found.